add cowstore.org

cosmosgenius opened the pull request:
`mail.cowstore.org` allows you to generate temp mail under the domain `cowstore.org`

FGRibreau commented:
Hello @cosmosgenius !

Could you please rebase and read our new way of updating `list.txt` (not `list.json`)? Thanks!

cosmosgenius commented:
Hey i have updated the PR. Sorry for the delay got caught up. I have removed the value from list.json to list.txt. 

There is a new line being added by the editorconfig settings as well. Let me know if that needs to be remove.

cosmosgenius commented:
@FGRibreau Hey just checking. Is there anything else required? Do let me know.
